tests/transforms/test_tape_expand.py

```diff
@@ -363,9 +363,9 @@ def custom_rot(phi, theta, omega, wires):
 
 
 # Decompose a template into another template
-def custom_basic_entangler_layers(weights, wires):
+def custom_basic_entangler_layers(weights, wires, **kwargs):
     return [
-        qml.AngleEmbedding(weights, wires=wires),
+        qml.AngleEmbedding(weights[0], wires=wires),
```

Comment:
This was not being properly tested before; `BasicEntangler` has weights with dim 2, whereas `AngleEmbedding` takes weights with dim 1